SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -72,7 +72,7 @@ When `ServerTravel` is triggered, the server tells all clients to begin to [`Cli
 To use `ServerTravel` with the GDK there are a couple of extra steps to ensure the SpatialOS deployment is in the correct state when transitioning maps. 
 
 #### Generate snapshot 
-Generate a snapshot for the map you intend to server transition to using the [snapshot generator]({{urlRoot}}/content/spatialos-concepts/generating-a-snapshot) and save it to `<GameRoot>\Content\Spatial\Snapshots\`. You can either copy your generated snapshot manually (from `<ProjectRoot>\spatial\snapshots\default.snapshot`) or set up your project settings to generate the snapshot for your level into that folder. You can find these settings via **Edit** > **Project Settings** > **SpatialOS Unreal GDK** > **Snapshot path**.
+Generate a snapshot for the map you intend to server transition to using the [snapshot generator]({{urlRoot}}/content/how-to-use-snapshots) and save it to `<GameRoot>\Content\Spatial\Snapshots\`. You can either copy your generated snapshot manually (from `<ProjectRoot>\spatial\snapshots\default.snapshot`) or set up your project settings to generate the snapshot for your level into that folder. You can find these settings via **Edit** > **Project Settings** > **SpatialOS Unreal GDK** > **Snapshot path**.
```

Comment:
Ah right, it's more of an ordering problem then. What about something like this?

> Generate a snapshot for the map you intend to server transition to, and save it to `<GameRoot>\Content\Spatial\Snapshots\`. To do this, you can either:
> * generate a snapshot (on the GDK toolbar, click **Snapshot**) and then copy the snapshot manually from `<ProjectRoot>\spatial\snapshots\default.snapshot`, or 
> * set up your project settings to generate the snapshot for your level into `<GameRoot>\Content\Spatial\Snapshots\`, and then generate the snapshot (on the GDK toolbar, click **Snapshot**)

Reply:
> Ah right, it's more of an ordering problem then. What about something like this?
> 
> > Generate a snapshot for the map you intend to server transition to, and save it to `<GameRoot>\Content\Spatial\Snapshots\`. To do this, you can either:
> > 
> > * generate a snapshot (on the GDK toolbar, click **Snapshot**) and then copy the snapshot manually from `<ProjectRoot>\spatial\snapshots\default.snapshot`, or
> > * set up your project settings to generate the snapshot for your level into `<GameRoot>\Content\Spatial\Snapshots\`, and then generate the snapshot (on the GDK toolbar, click **Snapshot**)

`select` rather than `click`, but looks good to me! :D Thanks!